### PR TITLE
Added Merge section to section

### PR DESCRIPTION
**Pull request by jeevatkm**
PR for #32

### CODE REVIEW

**Comment by jeevatkm** [comment]
It seems Travis CI failed for Go 1.4 with error [#58.1](https://travis-ci.org/brettlangdon/forge/builds/142605319)

```bash
package github.com/golang/lint/golint
	imports go/types: unrecognized import path "go/types"
make: *** [deps] Error 1
The command "make deps" failed and exited with 2 during .
```

**Comment by brettlangdon** [comment]
@jeevatkm thanks for the PR, I gave it a quick glance, but unfortunately do not have the time to fully review right now. I am hoping a little later today or tonight I can dig into it.

You can safely ignore the travis error, one of the dependencies used for testing `github.com/golang/lint` no longer supports Go 1.4. I will update `master` to remove `1.4` from the `.travis.yml`.

**Comment by jeevatkm** [comment]
Sounds good! Thanks.

**Comment by jeevatkm** [comment]
@brettlangdon when you get a chance please review and let me know.

File: section_test.go [hunk 1]
```diff
@@ -30,3 +32,122 @@ func TestSectionKeys(t *testing.T) {
 		t.Error("expected 'key3' to be in the list of keys")
 	}
 }
+
+func TestMergeSection(t *testing.T) {
+	config1Str := `
+	global = "global value";
```
Comment (brettlangdon) [inline section_test.go]: There are some indenting inconsistencies in these two config values, would you be able to fix those up?

**Comment by brettlangdon** [comment]
@jeevatkm sorry that I let this get away from me until now.

Other than the weird indenting from the test config strings, it looks good to me.

File: section_test.go (hunk 1 above)
Comment (jeevatkm) [inline section_test.go]: I think, you mean space before config lines correct?

**Comment by jeevatkm** [comment]
@brettlangdon Thanks for your review!

**Comment by jeevatkm** [comment]
@brettlangdon taken care, thanks.

**Comment by brettlangdon** [comment]
Yeah, much better! :+1:

Thank you for your contribution.

**Pull request merged**